azure-pipelines.yml

```diff
@@ -2,6 +2,7 @@ pool:
   vmImage: 'macos-latest'
 steps:
 - task: DownloadGitHubRelease@0
+  displayName: 'Download MacOS Server'
```

Comment:
```suggestion
  displayName: 'Download macOS Server'
```

Reply:
doesn't title casing supercede product names? honestly have no idea haha

Reply:
the pipeline name on azure is `Mac` so we should change one of them at least

Reply:
I think product names always take precedence.

Reply:
Alright, whoever has access to Azure should change the pipeline to `Jellyfin Server macOS` to match this then.

Reply:
@dkanada Will do.